cfg/param.yaml

```
@@ -4,4 +4,4 @@ depthimage_to_laserscan:
     range_min: 0.45
     range_max: 10.0
     scan_height: 1
-    output_frame: camera_depth_optical_frame
+    output_frame: "camera_depth_optical_frame"
```

Actually, sorry, while we are here, we should make this match what is actually in the code:

```suggestion
    output_frame: "camera_depth_frame"
```